modules/sql-engine/src/integrationTest/sql/aggregate/group/test_group_by_distinct.test

```diff
@@ -29,20 +29,20 @@ SELECT val FROM integers GROUP BY ALL GROUPING SETS ((val), (val))
 4
 
 query R rowsort
-SELECT val FROM ( VALUES (1), (1.0::REAL), (1.0::REAL), (2::DOUBLE), (1.1::REAL), (2::DECIMAL(2)) ) t(val) GROUP BY ALL GROUPING SETS ((val), (val))
```

Comment:
`1.1` does not have the exact representation as a double (we get `1.100000023841858`) , use `1.5` instead